tests/ProportionalControllerTest.cpp

```diff
@@ -0,0 +1,76 @@
+/**
+ * @file ProportionalControllerTest.cpp
+ * @author Giulio Romualdi
+ * @copyright Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ * @date 2020
+ */
+
+// Catch2
+#include <catch2/catch.hpp>
+
+#include <LieGroupControllers/ProportionalController.h>
+using namespace LieGroupControllers;
+
+TEST_CASE("Proportional Controller [SO(3)]")
+{
+    manif::SO3d desiredState, state;
+    desiredState.setRandom();
+    state.setRandom();
+
+    constexpr double dT = 0.01;
+    constexpr double kp = 10;
+    constexpr std::size_t numberOfIteration = 1e3;
+
+    ProportionalControllerSO3 controller;
+    controller.setGain(kp);
+    controller.setDesiredState(desiredState);
+
+    auto feedForward = Eigen::Vector3d::Zero();
+    controller.setFeedForward(feedForward);
+
+    for (std::size_t i = 0; i < numberOfIteration; i++)
+    {
+        controller.setState(state);
+        controller.computeControlLaw();
+        auto controlOutput = controller.getControl();
+        decltype(controlOutput) controlOutputDT = controlOutput.coeffs() * dT;
+        state = controlOutputDT + state;
```

Comment:
Is this the `lplus` operation under the hood? Could you please add a line of documentation?

Reply:
It is indeed `lplus` (`w (+) X`).

Reply:
Yep. 1b9471654491eb6cb5030150a6d3b1ec25b21bb0 I improved the documentation